src/sources/languages.ts

```diff
@@ -14,7 +14,7 @@ export default function getLanguages(): string[][] {
     // Starting from Chromium 86, there is only a single value in `navigator.language` in Incognito mode:
     // the value of `navigator.language`. Therefore the value is ignored in this browser.
     if (!(isChromium() && isChromium86OrNewer())) {
-      result.push(n.languages)
+      result.push(n.languages as string[])
```

Comment:
Thanks for letting us know! But ignoring TypeScript warnings isn’t a good solution. Please make a writable copy of the array instead:

```js
result.push([...n.languages])
```